src/main/java/seedu/address/logic/commands/LessonAddCommand.java

```diff
@@ -37,29 +35,25 @@
         + "Parameters: INDEX (must be a positive integer) "
         + "[" + PREFIX_RECURRING + "] "
         + PREFIX_DATE + "dd MMM yyyy "
-        + PREFIX_START_TIME + "HH:mm "
-        + PREFIX_END_TIME + "HH:mm "
+        + PREFIX_TIME + "HHmm-HHmm "
         + PREFIX_SUBJECT + "SUBJECT "
         + "[" + PREFIX_HOMEWORK + "HOMEWORK]...\n"
         + "Examples:\n"
         + "Makeup lesson: " + COMMAND_WORD + " 1 "
         + PREFIX_DATE + "10 Oct 2021 "
-        + PREFIX_START_TIME + "14:30 "
-        + PREFIX_END_TIME + "15:30 "
+        + PREFIX_TIME + "1430-1600 "
         + PREFIX_SUBJECT + "Science "
         + PREFIX_HOMEWORK + "TYS Page 2 "
         + PREFIX_HOMEWORK + "Textbook Page 52\n"
         + "Recurring lesson: " + COMMAND_WORD + " 1 "
         + PREFIX_RECURRING + " "
         + PREFIX_DATE + "09 Dec 2021 "
-        + PREFIX_START_TIME + "10:30 "
-        + PREFIX_END_TIME + "12:30 "
+        + PREFIX_TIME + "1030-1230 "
         + PREFIX_SUBJECT + "Math "
         + PREFIX_HOMEWORK + "TYS Page 2 ";
 
     public static final String MESSAGE_ADD_LESSON_SUCCESS = "Added new lesson: %1$s\nfor Person: %2$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
-    public static final String MESSAGE_DUPLICATE_LESSON = "This lesson already exists for this person.";
     public static final String MESSAGE_CLASHING_LESSON = "This lesson clashes with an existing lesson.";
```

Comment:
Might need a `list` command to let users know the existing lessons so they can figure out the timing/dates/days that are clashing.